tests/storage/blockvolume_test.py

```diff
@@ -51,9 +53,43 @@
 ])
 
 
+@contextmanager
+def make_volume(size, format=sc.RAW_FORMAT, prealloc=sc.SPARSE_VOL):
+    img_id = make_uuid()
+    vol_id = make_uuid()
+    # TODO fix make_volume helper to create the qcow image when needed
+    with fake_env(storage_type='block') as env:
```

Comment:
Using fake_env is very likey to create wrong tests. This env try to fake a block device with file and fake lvm, and may give wrong results.

This should be really tested in blocksd_test, using real storage domain and running real createVolume() calls.

Reply:
I'm ok with adding a `blocksd_test`, but I'd still keep this one, it is consistent with the other tests in the module, checking size, allocation, max_size, and optimal_size with the same strategy and code style.